src/explore-education-statistics-admin/src/pages/publication/components/PublicationUnpublishedReleases.tsx

```diff
@@ -0,0 +1,98 @@
+import PublicationDraftReleases from '@admin/pages/publication/components/PublicationDraftReleases';
+import PublicationScheduledReleases from '@admin/pages/publication/components/PublicationScheduledReleases';
+import publicationService from '@admin/services/publicationService';
+import { ReleaseSummaryWithPermissions } from '@admin/services/releaseService';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import WarningMessage from '@common/components/WarningMessage';
+import { useQuery } from '@tanstack/react-query';
+import React, { useMemo } from 'react';
+
+interface Props {
+  publicationId: string;
+  onAmendmentDelete?: () => void;
+}
+
+export default function PublicationUnpublishedReleases({
+  publicationId,
+  onAmendmentDelete,
+}: Props) {
+  const { data: releases, isLoading, isSuccess, refetch } = useQuery(
+    ['publicationUnpublishedReleases', publicationId],
+    () =>
+      publicationService.listReleases<ReleaseSummaryWithPermissions>(
+        publicationId,
+        {
+          live: false,
+          pageSize: 20,
```

Comment:
What happens if there are more than 20 unpublished releases? Seems like they wouldn't all be shown as there's no pagination for these releases.

Reply:
Indeed, but it's not likely for this to ever happen. Think in prod, the most draft/scheduled releases is like 4/5. Will just bump it up to something unreasonably high anyway, just in case.